### DISCUSSION TITLE
[Checkbox] Triggering onChange on enable and disable ?

### DISCUSSION THREAD

**Discussion by prudho** [fomantic/Fomantic-UI#1728] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
I was wondering the utility of triggering the `onChange` callback when we use the `enable` and `disable` behaviors. In which case is this usefull, since there is the `onEnable` and `onDisable` callbacks ?

I can make a PR in the minute if you @fomantic/maintainers are willing to.

**Comment by lubber-de** [comment, marked as answer]
`onchange` should IMHO only be called when the check state changes. Even, if you disable a checked checkbox, it will still be checked (thus: did not change)
As you already pointed out there are already existing callbacks for enable/disable

**Comment by prudho** [comment]
Done by #1744.
